internal/dic/content_appengine.go

```diff
@@ -0,0 +1,31 @@
+// Copyright 2015 ikawaha
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// 	You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build appengine
+
+package dic
+
+import (
+	"strings"
+)
+
+func newContents(b []byte) [][]string {
```

Comment:
__[gometalinter]__
* [dupl] duplicate of content.go:51-59
* [deadcode] newContents is unused

<!-- Comment by [SideCI](https://sideci.com) -->